runtime/benchmark_test.go

```diff
@@ -26,7 +26,32 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func benchmarkCreateAndStopVM(b *testing.B, vcpuCount uint32, kernelArgs string) {
+func createAndStopVM(
+	ctx context.Context,
+	fcClient fccontrol.FirecrackerService,
+	request proto.CreateVMRequest,
+) error {
+	uuid, err := uuid.NewV4()
+	if err != nil {
+		return err
+	}
+
+	request.VMID = uuid.String()
+
+	_, err = fcClient.CreateVM(ctx, &request)
```

Comment:
I would suggest adding a deadline because I experience sporadic timeouts. Probably 3-4 seconds can be reasonable. 
```suggestion
clientDeadline := time.Now().Add(time.Duration(*deadlineMs) * time.Millisecond)
ctx, cancel := context.WithDeadline(ctx, clientDeadline)
```

Reply:
Thanks. Will do.

Reply:
Addressed in the last revision.